pkg/crc/api/api.go

```diff
@@ -30,6 +37,18 @@ func CreateApiServer(socketPath string) (CrcApiServer, error) {
 	return apiServer, nil
 }
 
+func getListener(socketPath string, listenT listenerType) (net.Listener, error) {
+	switch listenT {
+	case localListenerType:
+		l, err := createIPCListener(socketPath)
+		return l, err
+	case remoteListenerType:
+		return nil, fmt.Errorf("Unimplemented listener requested.")
+	default:
+		return nil, fmt.Errorf("Unimplemented listener requested.")
```

Comment:
We probably can get rid of the `listenerType` argument for now, and readd it when there is a patch series which needs it?

Reply:
Yeah, definitely as i was talking to @gbraad about the remote scenario! will remove this when removing wip.